content/docs/capabilities/audit-logs.mdx

```diff
@@ -129,3 +129,20 @@ The keys described below usually contain the relevant information when debugging
 The most confusing keys for new users to understand are likely `allow-why-false` and `deny-why-false`. To better understand them, we should first discuss how [Pomerium Policy Language](/docs/internals/ppl) (**PPL**) works.
 
 PPL allows a request to a route if the claim matches at least one **allow** policy rule, and matches zero **deny** policy rules. With that in mind, `allow-why-false` and `allow-why-true` will describe a situation where the request either does or not not meet the requirements of an **allow** block a policy applied to that route. Conversely, `deny-why-true` and `deny-why-false` will describe why a request did or did not match a **deny** block for a policy assigned to the route.
+
+## Common Uses for Audit Logs
+
+### Zero Usage Tracking
+
+To determine which users are considered active for the purposes of billing, user details can be seen in the authorization logs under the `user` and `email` keys.
+
+This information is also available in the postgres database used by the Pomerium Databroker:
+
+```postgres
```

Comment:
This should be `sql`, because prismjs doesn't have a `postgres` language definition. Instead, they overload `sql`. It doesn't display perfectly, but we could PR it and add the `#>>` operator if we want.